receiver/splunkhecreceiver/receiver.go

```diff
@@ -72,6 +77,12 @@ var (
 	errInternalServerError    = initJSONResponse(responseErrInternalServerError)
 	errUnsupportedMetricEvent = initJSONResponse(responseErrUnsupportedMetricEvent)
 	errUnsupportedLogEvent    = initJSONResponse(responseErrUnsupportedLogEvent)
+
+	metadataMap = map[string]string{
```

Comment:
are these 3 the only supported query params? can you check if we can pass host as well?